tpu/hptuning/resnet-hypertune/config_resnet_hypertune.yaml

```diff
@@ -0,0 +1,28 @@
+trainingInput:
+  scaleTier: CUSTOM
+  masterType: standard
+  workerCount: 1
+  workerType: cloud_tpu
+  hyperparameters:
+    goal: MAXIMIZE
+    enableTrialEarlyStopping: true
+    hyperparameterMetricTag: my_top_5_accuracy
+    maxTrials: 3
```

Comment:
lets put more trials in the config. The single round run won't benefit a lot from hyperparameter tuning.

Reply:
done.